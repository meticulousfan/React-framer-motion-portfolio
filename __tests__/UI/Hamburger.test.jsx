/**
 * @jest-environment jsdom
 */

import React from "react";
import { render } from "@testing-library/react";

import Hamburger from "../../src/components/UI/Hamburger.component";

describe("Hamburger", () => {
  it("Hamburger laster inn og kan vises", () => {
    const { container } = render(<Hamburger />);
    expect(container).toMatchSnapshot();
  });
});
