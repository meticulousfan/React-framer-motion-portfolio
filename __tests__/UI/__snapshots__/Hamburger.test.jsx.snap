// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hamburger Hamburger laster inn og kan vises 1`] = `
<div>
  <button
    class="flex flex-col w-16 rounded justify-center items-center group "
    data-cy="hamburger"
    data-testid="hamburger"
    type="button"
  >
    <span
      class="sr-only text-white text-2xl"
    >
      Hamburger
    </span>
    <span
      class="h-1 w-10 my-1 rounded-full bg-white transition ease transform duration-300 not-sr-only opacity-100 group-hover:opacity-100"
    />
    <span
      class="h-1 w-10 my-1 rounded-full bg-white transition ease transform duration-300 not-sr-only opacity-100 group-hover:opacity-100"
    />
    <span
      class="h-1 w-10 my-1 rounded-full bg-white transition ease transform duration-300 not-sr-only opacity-100 group-hover:opacity-100"
    />
  </button>
</div>
`;
